windows.js

```diff
@@ -25,9 +25,11 @@ export async function install(platform, ruby) {
   }
   const base = url.slice(url.lastIndexOf('/') + 1, url.length - '.7z'.length)
 
+  const drv = (process.env['RUNNER_WORKSPACE'] || 'C')[0]
```

Comment:
Should we use `GITHUB_WORKSPACE` instead, which an officially documented env var?
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables

Reply:
Ok.  Done.